src/qml/components/StorageLocations.qml

```diff
@@ -21,25 +21,28 @@ ColumnLayout {
         ButtonGroup.group: group
```

Comment:
Part of suggestion (2), assign an identifier to set `checked = true` on it when user cancels to set custom data dir.

```suggestion
        id: defaultDirOption
        ButtonGroup.group: group
```

Reply:
great suggestions... done